app/services/symphony_client.rb

```diff
@@ -20,7 +20,7 @@ def login(library_id, pin)
       password: pin
     })
 
-    JSON.parse(response.body)
+    JSON.parse(response)
```

Comment:
Huh, this is interesting.  I didn't realize that the JSON or HTTP libraries worked like that.

Reply:
I can't see how it does according to the http gem documentation, but it works..

Reply:
It makes me feel a little nervous that it was so unexpected.  While I suppose tests would break, it seems like this might be a potential point of confusion (particularly if we change http connection gems or JSON parsers).

I might poke at this for a few mins to see if I can figure out what is going wrong when we use `response.body`.